tools/tune/tdfb/bf_design.m

```diff
@@ -17,15 +17,15 @@
 mkdir_check('data');
 
 if length(bf.steer_az) ~= length(bf.steer_el)
-	error('The steer_az and steer_el lengths need to be equal.');
+    error('The steer_az and steer_el lengths need to be equal.');
```

Comment:
I think you should configure the editor for indent by 8 with tab. We don't have a rule for .m but that would be same as for C code.